runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/WoodcuttingTreesOverlay.java

```diff
@@ -87,6 +87,15 @@ private void renderAxes(Graphics2D graphics)
 		{
 			if (treeObject.getWorldLocation().distanceTo(client.getLocalPlayer().getWorldLocation()) <= 12)
 			{
+				// Check if this is a redwood tree at the farming patch
+				if (client.getObjectDefinition(treeObject.getId()).getImpostorIds() != null)
+				{
+					final Tree tree = Tree.findTree(client.getObjectDefinition(treeObject.getId()).getImpostor().getId());
+					if (tree != Tree.REDWOOD)
+					{
+						continue;
+					}
+				}
```

Comment:
I don't think you need to check that the found `tree` is not a redwood, generally checking whether it is null should be sufficient I think?
```suggestion
				final ObjectComposition treeComp = client.getObjectDefinition(treeObject.getId());
				if (treeComp.getImpostorIds() != null && Tree.findTree(treeComp.getImpostor().getId()) == null)
				{
					continue;
				}
```

Reply:
I'm pretty unfamiliar with how imposter objects work so I was trying to guard against somehow the transformed ID being equal to some other tree enum. If this can't happen then your edit is good (and truthfully was my original error check)